Docs cli and server (#3479)
RasaHQ/rasa: docs/evaluating-models.rst

.. note::

  Make sure your model file in ``models`` contains both models, ``core``

>   and ``nlu``. If it does not contain a Rasa NLU model, Rasa Core will load

Reviewer: ```suggestion
  and ``nlu``. If it does not contain an NLU model, Core will use
```
